### PR TITLE
polish: Widget sim background

### PR DESCRIPTION
**Pull request by cmaddox5**
**Asana task**: [📺▶️ : 🇵🇱 | Make "speech bubble" around widgets match Figma](https://app.asana.com/0/1185117109217413/1202728175921081/f)

This one was a doozy but I think I got it. Some of the CSS in the existing background was not right/not at all written well so I corrected what I saw. The angled background is done in the `:before` part of the CSS files.

- [ ] Tests added?

### CODE REVIEW

File: assets/css/v2/bus_eink/simulation.scss [hunk 1]
```diff
@@ -8,8 +8,9 @@
   .simulation__full-page {
     height: 480px;
     width: 180px;
-    margin-right: 24px;
+    margin-right: 72px;
```
Comment (hannahpurcell) [inline assets/css/v2/bus_eink/simulation.scss:11]: The gl_eink margin also needs to be 72px

File: assets/css/v2/bus_eink/simulation.scss [hunk 2]
```diff
@@ -22,23 +23,41 @@
...
+
+    &:before {
+      content: "";
+      position: absolute;
+      right: 100%;
+      height: 216px;
+      border-top: 50px solid transparent;
+      border-right: 70px solid #2e3f4d;
+      border-bottom: 50px solid transparent;
+    }
```
Comment (hannahpurcell) [inline assets/css/v2/bus_eink/simulation.scss:38]: A more systematic way of creating this triangle is taking the height of the projection (316px), dividing it by two, and setting the top and bottom borders to that value, with a height of `0px`. The border-right value will be how steep you want the triangle. Keep content, position, and right as-is. So for e-ink:
```
&:before {
    content: "";
    right: 100%;
    // NO HEIGHT
    border-top: 158px solid transparent;
    border-right: 128px solid #2e3f4d;
    border-bottom: 158px solid transparent;
}
```
 Borrowed from https://css-tricks.com/snippets/css/css-triangle/.

Comment (hannahpurcell) [reply assets/css/v2/bus_eink/simulation.scss:38]: For PreFare as another example, it looks like border-right would be approx `144px` and top/bottom would be `72px`.

**Review by hannahpurcell — COMMENTED** [review]
Couple of tweaks. Otherwise, projection looks good

File: assets/css/v2/bus_eink/simulation.scss (hunk 2 above)
Comment (cmaddox5) [reply assets/css/v2/bus_eink/simulation.scss]: Awesome! SO much better.

**Review by hannahpurcell — APPROVED** [review]
Yeah, that looks great!

**Pull request merged**